dmc2gym/__init__.py

```diff
@@ -1,52 +1,14 @@
-import gym
-from gym.envs.registration import register
+# MODIFIED BY Yawen Duan (https://github.com/kmdanielduan)
+# moving the whole register process to registration.py
 
+import gym
 
-def make(
-        domain_name,
-        task_name,
-        seed=1,
-        visualize_reward=True,
-        from_pixels=False,
-        height=84,
-        width=84,
-        camera_id=0,
-        frame_skip=1,
-        episode_length=1000,
-        environment_kwargs=None,
-        time_limit=None,
-        channels_first=True
-):
-    env_id = 'dmc_%s_%s_%s-v1' % (domain_name, task_name, seed)
+from dmc2gym.registration import register_env, register_suite
+from dmc2gym.wrappers import DMCWrapper
 
-    if from_pixels:
-        assert not visualize_reward, 'cannot use visualize reward when learning from pixels'
+__all__ = ["DMCWrapper", "register_suite", "register_env", "make"]
 
-    # shorten episode length
-    max_episode_steps = (episode_length + frame_skip - 1) // frame_skip
 
-    if not env_id in gym.envs.registry.env_specs:
-        task_kwargs = {}
-        if seed is not None:
-            task_kwargs['random'] = seed
-        if time_limit is not None:
-            task_kwargs['time_limit'] = time_limit
-        register(
-            id=env_id,
-            entry_point='dmc2gym.wrappers:DMCWrapper',
-            kwargs=dict(
-                domain_name=domain_name,
-                task_name=task_name,
-                task_kwargs=task_kwargs,
-                environment_kwargs=environment_kwargs,
-                visualize_reward=visualize_reward,
-                from_pixels=from_pixels,
-                height=height,
-                width=width,
-                camera_id=camera_id,
-                frame_skip=frame_skip,
-                channels_first=channels_first,
-            ),
-            max_episode_steps=max_episode_steps,
-        )
-    return gym.make(env_id)
+def make(domain_name, task_name, **kwargs):
```

Comment:
This doesn't do much, I'm not sure we even really need it.

Reply:
True!